content/cpp/concepts/math-functions/terms/isfinite/isfinite.md

```diff
@@ -0,0 +1,71 @@
+---
+Title: 'isfinite()'
+Description: 'Returns a value telling whether the argument is a finite value.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`isfinite()`** function returns a value telling whether the argument has a finite value.
+
+## Syntax
+
+```pseudo
+isfinite(x)
+```
+
+The `x` parameter is finite if it a floating-point number that is neither infinite or `NaN` (e.g., division by zero, square root of a negative value).
+
+If `x` is finite, the `isfinite()` function will return a non-zero value for `true`. Otherwise, it will return zero for `false`.
```

Comment:
```suggestion
If `x` is finite, the `isfinite()` function will return `true` for a non-zero value. Otherwise, it will return `false` for zero.
```

Reply:
@yangc95 I'm pretty sure that `isfinite()` returns 0 or a non-0 value. I tested with this code:

```cpp
#include<iostream>  
#include<math.h>
#include <cfenv>
using namespace std;  
int main()  
{ 
  int result_1, result_2;
  result_1 = std::isfinite(1.0/1.0);

  result_2 = std::isfinite(1.0/0.0);
  std::cout << "Result 1: " << result_1 << endl;
  std::cout << "Result 2: " << result_2 << endl;
} 
```

And got this result:

```shell
Result 1: 1
Result 2: 0
```